fomotograph.rb

```diff
@@ -15,4 +15,8 @@
 get '/erasurez' do
   content_type :json
   data.to_json
-end
\ No newline at end of file
+end
+
+not_found do
+  'Not Found'
+end
```

Comment:
The current behavior echoes the request URL back to the client. That's a risky security practice because it can enable content injection xss attacks.

Reply:
Any changes will probably require content updates